src/utilities.js

```diff
@@ -250,3 +251,92 @@ export function getResults(request) {
     });
   };
 }
+
+export function getTrendingResults(request) {
+  return (searchType, obj) => {
+    const map = {
+      'Daily trends': {
+        path: '/trends/api/dailytrends',
+        extraParams: {
+          ed: convertDateToString(obj.trendDate, false, true),
+          ns: obj.ns,
+        },
+      },
+      'Real time trends': {
+        path: '/trends/api/realtimetrends',
+        extraParams: {
+          fi: 0,
+          fs: 0,
+          ri: 300, // # of trending stories IDs returned
```

Comment:
Is 300 an arbitrary selection?

Reply:
Well no it would appear this is the number of trending story IDs, however how to use these IDs to pull back data is a big mystery. I can make this a parameter, or in light of the fact that this appears to be unusable perhaps we can pair it back to reduce the data transfer bandwidth. The response size wouldnt change by too much the IDs are pretty short(40 chars) and in an array.

Reply:
Yeah these ids come back in an array. Google's UI for the real-time trends feature somehow can resolve these to trending stories (actual URLs), I can't see/figure out how the IDS are used to get the URLs. On top POST calls are used to get these URLS. So for now these IDs are useless. We can probably save a little in response size by utilizing a smaller number. One ID is about 40 characters in length.

Reply:
I tested this and it works as expected between 20 and 300. Anything outside this scope causes an error. The official webpage that uses this URL uses 300, and since for now we can't make use of the IDs, I'm leaving this hard coded for now to return the maximum (300).